dir.props

```diff
@@ -487,18 +487,10 @@
 
   <!-- Default Test platform to deploy the netstandard compiled tests to -->
   <PropertyGroup>
-    <TestTFM Condition="'$(TestTFM)'=='' and ('$(TargetGroup)'=='netstandard1.7' or '$(TargetGroup)'=='netcoreapp1.1')">netcoreapp1.1</TestTFM>
```

Comment:
This looks unrelated to the XML changes. Is there a reason you needed to move these to dir.targets as part of your work?